CMakeLists.txt

```diff
@@ -209,6 +209,28 @@ else()
   endif()
 endif()
 
+#-------------------------------------------------------------------------------
+# ROCm Default Target Configuration.
+#
+# ROCm does not have a stable instruction set like NVIDIA PTX; it requires
+# binaries specific to a target chip. In IREE, we can have various tests that
+# generates and runs the final deployable code. So for such tests we need to
+# make sure we give it the proper target chip and device bitcode path to link
+# against.
+#-------------------------------------------------------------------------------
+
+set(IREE_ROCM_TEST_TARGET_CHIP \"\" CACHE STRING
+  "Target chip for ROCm tests that need to compile device code. \
+   Defaults to \"\" to disable tests.")
```

Comment:
Should these use "HIP" instead of "ROCm"? I'm not clear on the distinction, but we should be consistent.

Reply:
Yeah changed to HIP.